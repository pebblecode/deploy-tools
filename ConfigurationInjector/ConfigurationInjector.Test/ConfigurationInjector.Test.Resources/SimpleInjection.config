<?xml version="1.0" encoding="utf-8"?>
<configuration>

  <appSettings>
    <add key="db.host" value="" />
    <add key="db.name" value="" />
    <add key="db.port" value="" />
    <add key="db.user" value="" />
    <add key="db.password" value="" />
  </appSettings>

</configuration>
