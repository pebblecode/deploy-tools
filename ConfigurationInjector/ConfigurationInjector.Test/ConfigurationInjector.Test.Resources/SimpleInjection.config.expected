<?xml version="1.0" encoding="utf-8"?>
<configuration>

  <appSettings>
    <add key="db.host" value="localhost" />
    <add key="db.name" value="sgp" />
    <add key="db.port" value="3400" />
    <add key="db.user" value="gambler" />
    <add key="db.password" value="g@mbl3r" />
  </appSettings>

</configuration>
